docs/supported_docker_environment/index.md

```diff
@@ -60,6 +60,34 @@ export TESTCONTAINERS_RYUK_DISABLED=true
     Previous to version 1.19.0, `export TESTCONTAINERS_RYUK_PRIVILEGED=true`
     was required for rootful mode. Starting with 1.19.0, this is no longer required.
 
+## Rancher Desktop
+
+In order to run testcontainers against [Rancher Desktop](https://rancherdesktop.io/) the env vars below should be set
+
+If you're running Rancher Desktop as an administrator in a MacOS (M1) machine:
+
+Using QEMU emulation
+
+```bash
+export TESTCONTAINERS_HOST_OVERRIDE=$(rdctl shell ip a show rd0 | awk '/inet / {sub("/.*",""); print $2}')
```

Comment:
on M1 with QEMU selected, I'm not able to get this value

Reply:
Hello, thank you for the review! With QEMU there is an additional requirement where you must allow admin access in Rancher Desktop (which can be set via the [Preferences > Application > General](https://docs.rancherdesktop.io/ui/preferences/application/general) dialog) and then the setting can be exported. If you wish to not allow admin access, then the VZ emulation must be used, along with the VZ export settings. I hope this clarifies and please let me know if there are any more questions.

Reply:
Thanks! I was missing that step.

Reply:
No worries!